system/system.proto

```diff
@@ -275,11 +275,37 @@ message TracerouteResponse {
   repeated int32 as_path = 12;   // AS path.
 }
 
+// RemoteDownload defines the details for a device to initiate a download of a
+// package from a remote location.
+message RemoteDownload {
+  // The path information containing where to download the package from.
+  // For HTTP(S), this will be the URL (i.e. foo.com/file.tbz2).
+  // For SFTP and SCP, this will be the address:/path/to/file
+  // (i.e. host.foo.com:/bar/baz).
+  string path = 1;
+
+  enum DownloadType {
+    DL_UNKNOWN = 0;
```

Comment:
I see that RemoteDownload is only used as part of Request and never a Response. In what situation would you use a DL_UNKNOWN value? Perhaps better to just use SFTP as default?